packages/pf-codemods/README.md

```diff
@@ -46,6 +46,396 @@ These rules are based off the breaking change notes for React. Each rule links t
 
 Some rules will add either a comment (`/* data-codemods */`) or data attribute (`data-codemods="true"`) in order to prevent certain other rules from applying an unnecessary fix.
 
+### accordionContent-remove-isHidden-prop [(#9876)](https://github.com/patternfly/patternfly-react/pull/9876)
+
+The `isHidden` prop has been removed from AccordionContent, as its visibility will now be set automatically based on the `isExpanded` prop on AccordionItem.
+
+#### Examples
+
+In:
+
+```jsx
+import { AccordionContent } from "@patternfly/react-core";
+
+export const AccordionContentRemoveIsHiddenPropInput = () => (
+  <AccordionContent isHidden />
+);
+```
+
+Out:
+
+```jsx
+import { AccordionContent } from "@patternfly/react-core";
+
+export const AccordionContentRemoveIsHiddenPropInput = () => (
+  <AccordionContent  />
+);
+```
+### accordionItem-warn-update-markup [(#9876)](https://github.com/patternfly/patternfly-react/pull/9876)
```

Comment:
Nit: maybe an empty line between the end of one rule and the heading of the next rule. Things blend a bit more together when the previous rule has no examples (`drawerHead-warn-updated-markup `).